pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools",
+    "wheel",
+    "Cython>=0.28.5",
+    "numpy>=1.13.3",
```

Comment:
not the `min-required-numpy` thingy?

Reply:
I didn't want to add it in this PR. I prefer putting things that can give rise to a long discussion in separate PRs :)